nats-streaming-server.go

```diff
@@ -127,7 +127,7 @@ func parseFlags() (*stand.Options, *natsd.Options) {
 	flag.StringVar(&stanOpts.ClientCert, "tls_client_cert", "", "Path to a client certificate file")
 	flag.StringVar(&stanOpts.ClientKey, "tls_client_key", "", "Path to a client key file")
 	flag.StringVar(&stanOpts.ClientCA, "tls_client_cacert", "", "Path to a client CA file")
-	flag.BoolVar(&stanOpts.EmbedNATS, "embed", true, "Embed the NATS Server")
+	flag.BoolVar(&stanOpts.EmbedNATS, "embed_nats", stand.DefaultEmbedNATS, "Embed the NATS Server")
```

Comment:
Makes sense to me. Thanks.

On Thu, Jul 14, 2016 at 10:54 AM, Ivan Kozlovic <notifications@github.com>
wrote:

> In nats-streaming-server.go
> <https://github.com/nats-io/nats-streaming-server/pull/125#discussion_r70841640>
> :
>
> > @@ -127,7 +127,7 @@ func parseFlags() (*stand.Options, *natsd.Options) {
> >  	flag.StringVar(&stanOpts.ClientCert, "tls_client_cert", "", "Path to a client certificate file")
> >  	flag.StringVar(&stanOpts.ClientKey, "tls_client_key", "", "Path to a client key file")
> >  	flag.StringVar(&stanOpts.ClientCA, "tls_client_cacert", "", "Path to a client CA file")
> > -	flag.BoolVar(&stanOpts.EmbedNATS, "embed", true, "Embed the NATS Server")
> > +	flag.BoolVar(&stanOpts.EmbedNATS, "embed_nats", stand.DefaultEmbedNATS, "Embed the NATS Server")
>
> There is a different set of config params for the Streaming and NATS
> server. I was referring to the user/pass for the NATS Server. When later we
> have streaming clustering and if we need user/pass specific to the
> streaming side of things, then the params will have to be named
> differently. Should be ok.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/nats-io/nats-streaming-server/pull/125/files/a1d758fd084007e2f30e194bad2f7525e8236061..1b7ca0d88480c871cf6e334a647ed6194844fc21#r70841640>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGq4LJ6hUhg65Q1LHD4bjqOAZzRDo2YOks5qVmmugaJpZM4JLmMr>
> .
>



-- 
*Larry McQueary* | Director, Messaging Technology
larry@apcera.com | @mcqueary | github.com/mcqueary